ftplugin/php/phpfolding.vim

```diff
@@ -181,10 +181,10 @@ function! s:PHPCustomFolds() " {{{
 	"       'g:searchEmptyLinesPostfixing'..
 
 	" Fold function with PhpDoc (function foo() {})
-	call s:PHPFoldPureBlock('\(^\|\s\)function\s\+[A-Za-z_][A-Za-z0-9_]*\s*(', s:FOLD_WITH_PHPDOC)
+	call s:PHPFoldPureBlock('^\(public\|private\|protected\|static\|\s\)*function\s\+[A-Za-z_][A-Za-z0-9_]*\s*(', s:FOLD_WITH_PHPDOC)
 
 	" Fold abstract function with PhpDoc
-	call s:PHPFoldProperties('^\s*\(abstract\)\s\(\(private\)\|\(public\)\|\(protected\)\s\)*\(static\s\)*function\s\+[A-Za-z_][A-Za-z0-9_]*\s*(', ";", s:FOLD_WITH_PHPDOC, 1, 1)
+	call s:PHPFoldProperties('^\s*abstract\(public\|private\|protected\|static\|\s\)*function\s\+[A-Za-z_][A-Za-z0-9_]*\s*(', ";", s:FOLD_WITH_PHPDOC, 1, 1)
```

Comment:
Here the brackets were not necessary and even misleading. Again the `static` and visibility keywords can appear now on any order (which is valid PHP syntax).